mat64/dense.go

```diff
@@ -104,10 +116,16 @@ func (m *Dense) isZero() bool {
 	return m.mat.Stride == 0
 }
 
+// Dims returns the number of rows and columns in the matrix.
 func (m *Dense) Dims() (r, c int) { return m.mat.Rows, m.mat.Cols }
 
+// Caps returns the number of rows and columns in the backing matrix. This
```

Comment:
I could say it's missing a '!' and that it's an emphatic "This!", But no. I was going to add an explanation and then decided against it.